attestation-agent/attester/Cargo.toml

```diff
@@ -37,6 +38,7 @@ hyper = { version = "0.14", features = ["full"], optional = true }
 hyper-tls = { version = "0.5", optional = true }
 tokio = { version = "1", features = ["full"], optional = true }
 tempfile = { workspace = true, optional = true }
+env_logger = { workspace = true, optional = true }
```

Comment:
Let's move the dep to follow alphabet order for better maintaince

Reply:
dropped this commit completely